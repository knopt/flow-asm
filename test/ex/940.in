1
37
-3.944541746368899
-2.515630899158574 
7.564882850791914 
-5.446594352090555 
5.648140027908944 
-8.349622561510836 
2.295428780818135 
6.0984558851612825 
5.306653897722061 
-5.80859127461869 
-4.898793869327771 
0.6324938752581701 
-3.766404868193753 
-4.706869370005917 
7.6839125853942 
-5.434427067934571 
1.59008292811661 
-3.894475086073383 
3.6990744706280925 
-7.644478451637569 
-3.456264951309249 
1.5258174155968778 
-6.9153606457790495 
-0.2596821604490529 
4.286261367073422 
-3.991002885479162 
-0.4592437176561148 
-9.007832092612336 
-2.830079925856685 
-6.818732320066935 
-3.4285027347257184 
-1.7611656844491996 
1.9025624442248414 
-7.887527850498676 
-9.300200316049159 
-4.275466033292259 
1.469847028289962 
-2.651855620205678 

-4.718624500947826 4.453892860038813 -0.3320990016417227 4.215311774816978 9.690701389422166 6.154164009457158 -8.795518340371846 0.36089195416629316 -7.8026988700666955 -6.993017510451402 3.9433971167030197 7.718611132813553 -7.601192588172873 -6.994244694286971 3.962537405459919 2.7188626088932555 1.1774538146354185 3.470526215878607 8.67938346782475 7.037516924077714 5.401223052694274 2.921772143770978 -3.7577327390814963 3.3142237289826078 -0.6228876462745276 -2.907896777534413 -6.101805585569449 -9.355522494040464 1.9961134133707326 3.65885368204653 -8.207582899214712 -6.082777586093369 5.955424049158937 -7.2168139974760726 -4.67373362490876 0.49261122193067663 -6.847441444864062 
