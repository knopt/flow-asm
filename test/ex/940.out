-61.08041540190669 
190.52641719008463 
-197.87356989374499 
117.92164315669689 
-283.48874144540576 
28.640879215556915 
105.38517158345815 
172.87366732885732 
-65.03979848368276 
-38.29151983533385 
28.870195628735036 
-77.99662372084424 
-85.86592789589142 
241.17616703962364 
-147.96368281181282 
38.74949236605748 
-130.6261933646606 
69.60038508290941 
-235.0576779052977 
-130.78917752970457 
11.940198721725682 
-166.30719615490023 
-4.786338302895687 
109.79639660200313 
-96.94598852458762 
79.7538268449741 
-101.24941020095466 
56.87994524385534 
-102.01836795677352 
-27.13519809913223 
11.459434083256827 
110.36342654806064 
-105.30127541845559 
-121.3365624574042 
-12.752445914377864 
101.28695180067608 
-14.763901076357044 
